mitodl/mit-learn
# Clean out the static frontend

### What are the relevant tickets?





https://github.com/mitodl/hq/issues/5846

### Description (What does it do?)


Removes the ./frontends/mit-learn workspace and any references. This was the entrypoint for the previous static frontend, now replaced by the Next.js entrypoint at ./frontends/main.

Also removes the E2E/browser tests as these have been inactive and have not been migrated to work with Next.js.


### How can this be tested?


Generally, all tests should pass on frontend and backend and services should build and run with Docker without error.
